enterprise/admin/__init__.py

```diff
@@ -101,6 +101,22 @@ def identity_provider(instance):
         ec_idp = instance.enterprise_customer_identity_provider
         return ec_idp and ec_idp.provider_name or ec_idp.provider_id
 
+    def enterprise_catalog(self, instance):
+        """
+        Instance is EnterpriseCustomer.
+
+        Return catalog id with catalog details link to display in enterprise customer list view.
+        """
+        catalog_url = get_catalog_admin_url_template().format(catalog_id=instance.catalog)
+        if instance.catalog:
+            return "{catalog_id}: <a href='{catalog_url}' target='_blank'>View catalog details.</a>".format(
```

Comment:
This seems sort of odd -- in one case we return hyperlink markup, and in another we return...None?  I think I would rather see this method turned around to instead use a guard clause:

```
if not instance.catalog:
    return None
[...now perform actual work]
```